cirq-google/cirq_google/optimizers/two_qubit_gates/example.py

```diff
@@ -1,104 +0,0 @@
-# pylint: disable=wrong-or-nonexistent-copyright-notice
-"""Example application of the two-qubit gate compilation algorithm.
```

Comment:
I think we should probably keep this example around; there's nothing deprecated in here. Not sure what's the best place to move this to; Maybe Cirq/examples?

Reply:
Thanks for the catch!